lib/kubernetes-deploy/kubernetes_resource/pod_disruption_budget.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class PodDisruptionBudget < KubernetesResource
+    TIMEOUT = 30.seconds
```

Comment:
I copied the 30 seconds from the ConfigMap which I considered to be very similar.

Maybe both should be changed?